en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md

```diff
@@ -0,0 +1,215 @@
+    | **keyStoreLocation**   | Name of the keystore file. **The file must be located in the `<API-M_ANALYTICS_HOME>/resources/security` directory.** |
+    | **keyStorePassword**   | Keystore password                                                |
+
+2. Update the HTTP transport configurations.
+   
+    This is used for authentication, encryption, and signing with regard to the analytics dashboard web application.
+
+    ```yaml
+    wso2.transport.http:
+      ...
+      listenerConfigurations:
+        - id: "default-https"
+          ...
+          keyStoreFile: "${carbon.home}/resources/security/wso2carbon.jks"
+          keyStorePassword: wso2carbon
+          certPass: wso2carbon 
+    ```
+    The elements in the above configuration are described below:
+      
+    | Element                | Description                                                      |
+    |------------------------|------------------------------------------------------------------|
+    | **keyStoreFile**       | Name of the keystore file. **The file must be located in the `<API-M_ANALYTICS_HOME>/resources/security` directory.** |
+    | **keyStorePassword**   | Keystore password                                                |
+    | **certPass**           | Private key password                                             |
+   
+3. Update the Secure Vault configurations.
+   
+     1. Update the configurations in the `deployment.yaml` file.
+    
+         The Secure Vault configurations are used to store encrypted passwords that are mapped to aliases.
+
+         ```yaml
+         wso2.securevault:
+           secretRepository:
+             ...
+             parameters:
+               privateKeyAlias: wso2carbon
+                keystoreLocation: ${sys:carbon.home}/resources/security/securevault.jks
+               ... 
+         ```
+        The elements in the above configuration are described below:
+          
+        | Element                | Description                                                      |
+        |------------------------|------------------------------------------------------------------|
+        | **privateKeyAlias**    | Private key alias                                                |
+        | **keyStoreLocation**   | Name of the keystore file. **The file must be located in the `<API-M_ANALYTICS_HOME>/resources/security` directory.** |
+
+    2. Provide the keystore and private key password in the Base64 encoded format in the `<ANALYTICS_HOME>/conf/dashboard/master-keys.yaml` file.
+
+        ??? info "Click here for more info on the `master-keys.yaml` file"
+            The base64 encoded plaintext keystore password and the base64 encoded plaintext private password of the keystore used for Secure Vault is defined in the master-keys.yaml file. WSO2 Carbon Server reads this file to obtain these two passwords, so that it can decrypt the passwords in the `secrets.properties` file in order to start the server.
```

Comment:
plain text -> two words?

Reply:
Shall we reword this to, "The keystore password and private password which is in plain text is base64 encoded and stored in the master-keys.yaml"

Reply:
Fixed.
Changed to 
"The keystore password and private password used for Secure Vault, which is in plain text is base64 encoded and stored in the `master-keys.yaml` file. "